src/spectro_inlets_quantification/sensitivity.py

```diff
@@ -1137,3 +1140,59 @@ def plot_F_vs_f(
             F_fit = self.alpha * f_fit
             ax.plot(f_fit, F_fit, "k--")  # type: ignore
         return ax
+
+
+def append_sensitivity_factors(*args):
+    """Recursively append elements in a list"""
```

Comment:
As much as I like re recursive (reductive) approach to this, I'm wondering if a iterative approach might not be easier to read (the other stuff isn't done that much anymore). You can also use a unpackagin/packing trick to split the list up:
```python
first, second, *remaining = args
result = append_two(first, second)
for sf in remaining:
    result  = append_two(result, sf)
return result
```

You could even unpack only the first and let that be result, and then the loop, so you will not need the extra add for the first two, but then you will different behavior for the special case of 1 argument.

Reply:
But I don't mind it as is, so view it as an alternative suggestion.

Reply:
I agree the unpacking trick is a bit simpler and much easier to read! Will use it next time - this code is not needed when it is easy to just use `+`.